pkg/project/project.go

```diff
@@ -1,7 +1,7 @@
 package project
 
 var (
-	bundleVersion        = "7.0.1-dev"
+	bundleVersion        = "9.9.9"
```

Comment:
Please don't change the version for testing. `-dev` version shouldn't be deployed anywhere and you should be fine to test with that.

Reply:
I was suggesting to change this value in order to test. I do not follow your reasoning. Let's chat tomorrow on how to properly do this. IMO this here is totally legitimate and I am not aware of a different way to do it.

Reply:
Since I wasn't aware of a different way either I'm currently testing it like this.
If there's a consensus on how to do it properly it would be great if we can document it so that there is less confusion next time.